### PR TITLE
native vs hybrid app article

### PR DESCRIPTION
**Pull request by ipitah**
native vs hybrid app article

#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.

**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process and style-once published, you can then submit any number of articles.)**
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

### CODE REVIEW

**Comment by ipitah** [comment]
This fixes https://github.com/section-io/engineering-education/issues/1188

**Comment by hectorkambow** [comment]
👋 Thank you for submitting this PR - we will review it ASAP - all PRs go into our queue will be looked over before being approved to be published.

**Comment by EKahuha** [comment]
@hectorkambow 
Can I review this one?

**Comment by hectorkambow** [comment]
@EKahuha  will be reviewing this article. 🚀

**Comment by ipitah** [comment]
@hectorkambow That well noted

**Review by EKahuha — COMMENTED** [review]
@ipitah Great comparison between native and hybrid applications. I enjoyed reading it. However, I have noted several grammatical mistakes, especially on punctuation. I have added suggestions on the first few paragraphs, but you'll need to run your article through a premium [Grammarly](https://app.grammarly.com/). Also, use [Hemmingway App](http://www.hemingwayapp.com/) to check the readability of your content. Improve the 'very hard to read sentences'.

File: articles/choose-native-vs-hybrid/index.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+1. **React native**
+It combines native with React to produce a JavaScript library that allows the creation of truly native applications that do not compromise users' experience and have cross-platform support.
+
+2. **Xamarin**
+It is a platform for native cross-platform mobile application development that extends the .NET developer platform with tools and libraries. It is used to develop applications for operating systems such as iOS, tvOS, macOS, and Windows.
+
+ #### Conclusion
+To decide which approach to follow, one should consider a couple of factors such as time, complexity, and resources. Applications that require intense features, are sophisticated, and have sufficient delivery time should use a native approach. There exist other languages that allow the development of cross-platform mobile applications with native behavior. This article has pointed out what developers, especially beginners, needed to make informed decisions. Start developing your mobile application today, happy coding!
+
+#### References
```
[the remark this answers is not in the document]
Comment (ipitah) [reply articles/choose-native-vs-hybrid/index.md:56]: This is well noted, will capture that in my future articles.

**Comment by ipitah** [comment]
@EKahuha Kindly, find the revised article.

**Review by EKahuha — APPROVED** [review]

**Review by EKahuha — APPROVED** [review]

**Comment by EKahuha** [comment]
@hectorkambow, this article is ready for final review.

**Comment by WanjaMIKE** [comment]
@EKahuha added final review label

**Comment by hectorkambow** [comment]
@ipitah nice overall article - please go over the changes made during final review to implement into future article and avoid multiple revisions and tiered payout.

cc @EKahuha  please do the same to help improve any future article revisions. 👍

**Pull request merged**